profile: Use processed image field

harshithpabbati opened the pull request.

vibhoothi reviewed (changes requested):
Hi,
Please add previous commit ID mentioning the hotfix for this

harshithpabbati committed:
profile: Use Processed Image Field

This fixes regression brought by #49a1206614350e283a1f4569ec3438592eea998e
Replaced file field with processed image field to validate the file size and to resize the images.

harshithpabbati commented:
@vibhoothiiaanand made the required changes. Check it out.

vibhoothi reviewed (approved):
LGTM
